docs/signals/concepts/index.md

```diff
@@ -4,80 +4,224 @@ sidebar_position: 1
 sidebar_label: "Concepts"
 ---
 
+Signals introduces a new set of data governance concepts to Snowplow. As with schemas for Snowplow event data, Signals components are strictly defined, structured, and versioned.
 
-## Entities
+The fundamental Signals component is the `Entity`. The Signals components attributes and interventions are defined relative to entities. Attributes for a given entity are grouped together into views and services, for ease of management and deployment.
 
-The foundation of Signals are entities (distinct from [Snowplow entities](/docs/fundamentals/entities/index.md), but they can be related).
+```mermaid
+flowchart TD
+    Entity --> Stream
+    Entity --> Batch
+    Entity --> Int1[**Intervention 1**]
+    Entity --> Int2[**Intervention 2**]
 
-An <dfn>entity</dfn> is the core unit that all the other Signals features relate to, and can be anything with an "identifier" that you can capture in a Snowplow event.
+    subgraph Stream
```

Comment:
I am not sure if this is technically correct – individual attributes have no relation to entities and also don't specify whether they should be processed in batch or stream. This is defined at the View level. I think it would be more correct to link the entities to views and also have the batch/stream distinction on the views, not on the attributes